cmake/projects/date/hunter.cmake

```diff
@@ -20,6 +20,17 @@ hunter_add_version(
     4ed983e1d19ee28bc565bd62907d203304b38cf7
 )
 
+hunter_add_version(
+    PACKAGE_NAME
+    date
+    VERSION
+    master-20200420
```

Comment:
Can you please change this to `2.4.1-e12095f`? That's the Hunter convention for packages versioned from master - last released version plus the first 7 of the commit hash.

Reply:
I didn't know this convention. Is it documented somewhere. I recall using version number plus `pN`, for example `2.4.1-p1`, and if we have to do another interim release `2.4.1-p2`

Reply:
@NeroBurner the `-pN` convention is for forked packages that have a patch applied.

Here's the documentation on versioning: https://hunter.readthedocs.io/en/latest/creating-new/patch.html

```
If you want to have version that is not released yet, say da39a3e, then create branch hunter-1.3.15-da39a3e. If there are not tags in upstream then start with dummy v0.0.0 version to avoid conflict with future releases, i.e. create branch hunter-0.0.0-da39a3e.
```

Reply:
Done